ion/marshal.go

```diff
@@ -38,6 +38,42 @@ type Marshaler interface {
 }
 
 // MarshalText marshals values to text ion.
+//
+// Different Go types can be passed to be Marshalled to their corresponding Ion types.
+//
+//	val, err := MarshalText(9)
+//	if err != nil {
+//		t.Fatal(err)
+//	}
+//
+// 	type root struct {
+//		A struct {
+//			B int `ion:"b"`
+//		}
+//		C int `ion:"c"`
+//	}
+//  v := root{A: {B: 6}, C: 7}
+//	val, err := MarshalText(v)
```

Comment:
Can we do fmt.Println(val) to show the Ion text

Reply:
great idea, added